params/safety_config.yaml

```diff
@@ -0,0 +1,11 @@
+safety_bonds:
```

Comment:
we need to have form_bond: false here on each node so that if a roslaunch session is closed without resetting rosparam the node monitor won't look for nodes that aren't there.